builder.go

```diff
@@ -224,6 +224,8 @@ type Builder struct {
 	PendingCopies  []Copy
 
 	Warnings []string
+
+	AsImageName []string
```

Comment:
Why is this an array?

Reply:
An individual builder is invoked for a given stage - perhaps it would be cleaner if you make this an executor responsibility (like dockerclient)?

Reply:
cut/paste error, it should be a string.

Reply:
I've repushed as a string for now, ty for spotting that.  

My first pass through I changed the name field to be 'name + " AS " + args[2] which eventually set the b.RunConfing.Image  to that value.  The executor then sniffed out the " AS " in the Image and handled it from there.  I thought that was a little funky and was concerned that it might mess up other folks using imagebuilder, so I went to this current approach which I thought made handling the AS part of stuff a bit cleaner, but did suffer the weight of the new field in the struct.

Had you other thoughts for how to get the executor to handle the AS clause?  Wouldn't be the first time I missed something obvious!